repositories.yaml

```diff
@@ -840,9 +840,14 @@ applications:
       - brizental@mozilla.com
     branch: main
     metrics_files:
-      - glean/metrics.yaml
+      - src/shared/telemetry/metrics.yaml
+      - src/shared/telemetry/metrics_deprecated.yaml
```

Comment:
Unfortunately we currently enforce that files are named `metrics.yaml`, see also #364 
Let me figure out if we can now just lift that check.